### PR TITLE
Remove hardcoded username

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
@edx/ecommerce

### CODE REVIEW

**Comment by rlucioni** [comment]
@MatthewPiatetsky why not just do this as part of #424?

**Comment by rlucioni** [comment]
Let me rephrase: it seems like #424 obviates the need for this.

**Comment by MatthewPiatetsky** [comment]
@rlucioni I put this up in case we decided we did not want to do that, or want to merge this earlier. Seems like merging anything is a moot point for now until we figure out what is happening with the stage catalog
